components/button/button.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Button|Sapota UI</title>
+    <link rel="stylesheet" href="../../docs/css/index.css" />
+    <link rel="stylesheet" href="./button.css" />
+  </head>
+  <body>
+    <div class="wrapper-btn">
+      <h2>Buttons</h2>
+      <p class="desc"></p>
+      <div class="container-btn">
+        <button class="btn btn-primary">Primary</button>
+        <button class="btn btn-secondary">Secondary</button>
+        <button class="btn btn-success">Success</button>
+        <button class="btn btn-danger">Danger</button>
+        <button class="btn btn-link">Link</button>
```

Comment:
button link should generally be an anchor element acting as a button.

Reply:
In my opinion, semantically it should be a button element with link style because we are styling for button component